PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3873,6 +3873,9 @@ void Notepad_plus::updateStatusBar()
 	_statusBar.setText(strLnColSel, STATUSBAR_CUR_POS);
 
 	_statusBar.setText(_pEditView->execute(SCI_GETOVERTYPE) ? TEXT("OVR") : TEXT("INS"), STATUSBAR_TYPING_MODE);
+	
+	//updating "Go To" dialog

```

Comment:
So the comment is OK, but really I think the line of code itself explains it pretty well.  In such cases maybe it is best to not use the comment.

Reply:
Maybe it isn't the best idea to call this function BEFORE the GoTo window has been created; it seems to work but: 
![image](https://user-images.githubusercontent.com/77065706/194705213-5abf47d1-8188-4dd8-ac8a-0cd962dd6d55.png)

Reply:
How about using 
` if (_goToLineDlg->isCreated() && _goToLineDlg->isVisible()){_goToLineDlg.updateLinesNumbers();}`
instead of
`_goToLineDlg.updateLinesNumbers();`

Reply:
It's your PR.  :)

Reply:
There I just updated the PR.

Reply:
So, do I just wait for review at this point?

Reply:
Yes.

Reply:
Note that review could entail rejection of the entire idea.  :-)

Reply:
Ok, understood. Thanks!